lib/app_jwt/app_jwt.c

```diff
@@ -14,7 +14,13 @@
 #include <date_time.h>
 #include <psa/crypto.h>
 #include <psa/crypto_extra.h>
+
+#if defined(CONFIG_BOARD_NATIVE_SIM)
+#define IAK_APPLICATION_GEN1 0x41020100
+size_t strnlen(const char *, size_t);
```

Comment:
why is this here?

Reply:
This is needed for the unit test and native_sim versions to compile. I cannot use the usual method to get strnlen, because some headers are not compatible with it and generate more compile warnings.